### ISSUE TITLE
GCwiki editor accessibility issue

### ISSUE THREAD

**Issue by marc-donofrio** [gctools-outilsgc/gcconnex#2224]
Subject: Flash

Description:
There is no reason to allow CKEditor to upload flash files.
 
Why would you allow either tables, divs or blocks in the body of this.
 
Simplify the WYSIWYG to options that you know are accessible. 
 
Also, as far as simplifying this form, why on earth is Organization mandatory?  
 
Not sure why Occupation should be either.

## For the development team
- [ ] Issue user story documented
- [ ] UX input received
- [ ] Design completed
- [ ] Design validated by business team / UX
- [ ] Code review completed by peer
- [ ] Issue closing comment references any duplicate or connected issues or pull requests
- [ ] Issue closed

**Comment by Steph4104** [comment]
For the upload flash file, base on the documentation, we can modify what type of file we allow but we need to have access to the code. https://www.mediawiki.org/wiki/Manual:$wgFileExtensions

**Comment by Steph4104** [comment]
GCWiki allow user to upload flash file via an upload button. There is no CKEditor on GCwiki or Pedia, only on Connex and Collab.

CKEditor on Collab and Connex has the flash button that let user link a flash file. Be able to remove this option will required a lot of work

Question is, do we want to remove option to upload flash file on GCWiki? No user using it on GCwiki but some user using it on Pedia. 

https://www.gcpedia.gc.ca/gcwiki/index.php?title=Special:MIMESearch&limit=500&offset=0&mime=application%2Fx-shockwave-flash

https://wiki.gccollab.ca/Special:MIMESearch?mime=application%2Fx-shockwave-flash

Why should we remove this option? We can provide information to the user that is not accessible. It will be the choice of the user to continu or not using it

**Comment by Phanoix** [comment]
It should be fine to remove it from wiki, in the olden days for gcpedia that was how users embedded videos in pages that they couldn't host elsewhere, but with wiki being external that's not an issue.
